how_to_select.md

```diff
@@ -0,0 +1,36 @@
+# Selecting an Image
+
+We have two Docker images followings:
+
+1. `rubydata/minimal-notebook`
+2. `rubydata/datascience-notebook`
+
+## Ruby
+
+These two images contains same Ruby components for data science followings:
+
+- pry
+- iruby
+- pycall
+- numpy
+- pandas
+- matplotlib
+- numo-narray
+- numo-linalg
+- nmatrix
+- nmatrix-lapacke
+- red-arrow
+- red-arrow-numo-narray
+- red-arrow-nmatrix
+- daru
+- rbplotly
+
+## `rubydata/minimal-notebook`
+
+`rubydata/minimal-notebook` consists of minimal components for data science using Ruby, Python.
```

Comment:
Please replace `Ruby, Python.` by `Ruby and Python.`

Reply:
Done.